go.mod

```diff
@@ -31,3 +32,6 @@ require (
 	k8s.io/client-go v0.20.4
 	sigs.k8s.io/yaml v1.2.0
 )
+
+// manifoldcos code is not thread safe
+replace github.com/manifoldco/promptui => github.com/1lann/promptui v0.8.1-0.20201231190244-d8f2159af2b2
```

Comment:
We are replacing github.com/manifoldco/promptui with a fork that contains a fix for thread safety (which is only required to run the tests).

Of course, this shouldn't be merged as it is, and we were thinking on either forking into rancher-sandbox, or skipping testing (we would prefer not to).

Reply:
I vote for substituting the library with the builtin reader. I could push this commit to the branch of this PR, if that's ok:
https://github.com/viccuad/hypper/commit/7fe8b3c7fbf41e877167d00dcbc0a7d5ff4ba41d

Reply:
cherry-picked into https://github.com/rancher-sandbox/hypper/pull/157/commits/b7b4c3e41b04c446e85968fdc84867b1c899e2b2.